docs/Contributing.md

```diff
@@ -0,0 +1,56 @@
+## Issue tracking
+
+`minnie-kenny.sh` issues are tracked on the [DSP
+Workbench](https://www.broadinstitute.org/data-sciences-platform/workbench) JIRA under [Batch
+Analysis](https://broadworkbench.atlassian.net/projects/BA/issues). You will need to sign up with a free account before
+being able to view or create issues.
+
+Several known issues are listed in the documentation under [Common Questions](../CommonQuestions/).
+
+## Code style
+
+`minnie-kenny.sh` is mostly formatted according to the [Google Shell Style
+Guide](https://google.github.io/styleguide/shell.xml).
+
+Exceptions to the original style guide:
+
+- Uses POSIX compatible `sh`
+- Uses an `.sh` extension
+- Lines are 120 characters
+- Uses a `-` in the source file name
+
+## Tests
+
+See [Testing](../Testing/) for a list of existing tests. Before any [pull requests](#pull-requests) will be accepted, all
+existing and new tests must pass, including `lint` and `coverage`. If your additions to `minnie-kenny.sh` are not
+covered then add a new test. If you are having trouble designing a coverage test please mention it in the Jira ticket
+covering your [issue](#issue-tracking) / [pull request](#pull-requests).
+
+## Documentation
+
+The documentation source files are available in
+[GitHub under /docs](https://github.com/broadinstitute/minnie-kenny/tree/master/docs). The documentation is built using
+[MkDocs](https://www.mkdocs.org), and may rendered locally using `mkdocs serve`.
+
+## Pull requests
+
+Updates are welcome and may be submitted via [GitHub pull
+requests](https://github.com/broadinstitute/minnie-kenny/pulls).
+
+Every pull request should refer to a Jira issue in the description. See above under [Issue Tracking](#issue-tracking)
+for information on Jira. The pull request description should contain the string "BA-#" where "#" is the Jira issue
+number, for example "My latest pull request BA-99999".
```

Comment:
As far as I can tell, the square brackets around Jira issue links started because one (or more?) people misread a confusing section of the Jira docs circa 2010, and then everyone else followed along...

[These old docs state](https://confluence.atlassian.com/display/APPLINKS030/Using+Link+Rendering#space-menu-link-content):

> You could then reference an issue in your JIRA project ... by just typing the issue key, e.g. [JRA-1234]

That sentence uses square brackets for some reason... but just below that line there's a table that explicitly mentions:

| To link to ...                     | Type this                       | Examples                  |
|------------------------------------|---------------------------------|---------------------------|
| an issue                           | \<Issue Key\>                   | _TEST-123_<br/>_ACME-456_ |
| a wiki page in the same project    | \[\<Page Name\>\]               | _\[My Draft Page\]_       |
| a wiki page in a different project | \[\<Space Key\>:\<Page Name\>\] | _\[TEST:My Draft Page\]_  |

In the current docs the examples are even clearer, [in this case for explicit commit messages](https://confluence.atlassian.com/display/APPLINKS060/Streamline+software+development+with+Atlassian+tools?preview=/831660181/831660193/Stream-commit-details3.png), where none of the example descriptions use square brackets yet still are detected.

**TL;DR I don't use brackets, as in this current PR, and linking still works.**